docs/source/cloud-setup/cloud-permissions/aws.rst

```diff
@@ -1,39 +1,385 @@
-
-.. _cloud-permissions-aws:
-
 AWS
 =====
 
+.. note::
+
+    By default, SkyPilot will use the credentials you have set up locally. For most cases, the :ref:`getting started instructions <aws-installation>` are all you need to do. The steps below are **optional advanced configuration options**, aimed primarily at cloud admins and advanced users.
+
+
+.. _aws-sso:
+
+Using AWS SSO
+-------------
+
+`AWS IAM Identity Center <https://aws.amazon.com/iam/identity-center/>`_ (Successor to AWS Single Sign-On, or SSO) is supported.
+
+.. _sso-feature-compat:
+
+SSO login has limited functionality *across multiple clouds*. If you use multiple clouds, you can :ref:`set up a dedicated IAM user and access key <dedicated-aws-user>` so that instances launched on other clouds can use AWS resources.
+
+.. list-table::
+   :header-rows: 1
+
+   * - *Supported features:*
+     - SSO credentials
+     - Static credentials
+   * - Use S3 buckets on an AWS cluster
+     - |:white_check_mark:|
+     - |:white_check_mark:|
+   * - Use S3 buckets on a cluster in another cloud
+     - |:x:|
+     - |:white_check_mark:|
+   * - Run :ref:`managed jobs <managed-jobs>` across multiple clouds
+     - |:yellow_circle:| [1]_
+     - |:white_check_mark:|
+
+.. [1] To allow managed jobs to run on AWS instances, make sure your controller is also on AWS, by :ref:`specifying the controller resources <jobs-controller-custom-resources>`.
+
```

Comment:
These are multicloud-related notes. Can we move it down, behind the instructions below? Can even be a subsection, "SSO and Cross-cloud Features"?

Reply:
I think it's pretty important for SSO users to see that some features won't work, otherwise it will just seem like SkyPilot is broken. We could leave a very short note here and link to a subsection with the details below, though.